tests/functions.sh

```diff
@@ -1,6 +1,6 @@
 print_array () {
     #usage: print_array "${my_array[@]}"
-    arr=("$@")
+    arr=( "$@" )
```

Comment:
remove space

Reply:
@edaena are you planning to revert this change?

Reply:
@andrebriggs reverted the change and also reverted the requirement for subscription id, it will just use the default one that's on `az login`